src/api/controllers/UserController.js

```diff
@@ -32,6 +32,26 @@ const authResponse = user => ({
 });
 
 module.exports = {
+  changePassword: (req, res) => {},
+
+  getProfile: (req, res) => {
+    User.findOne({ _id: req.params.userId })
```

Comment:
Please check if the userId matches the token's id before allowing access to getProfile, as per the [project board](https://github.com/din0s/ActionHeroes/projects/1).